spaceflights-pandas/hooks/post_gen_project.py

```diff
@@ -27,5 +27,5 @@ def main(selected_tools):
 
     # Execute the script only if the tool is selected.
     # This ensures the script doesn't run with kedro new --starter but only with the tools flow option.
-    if selected_tools != "none":
+    if selected_tools != "['None']":
```

Comment:
Looks like it should be:
'["None"]' instead of "['None']"?
to be consistent with:
[converting](https://github.com/kedro-org/kedro/pull/3462/files#diff-e3c3f6f5910e7f8d3133cfbeec750588c9328e0507c5464b6b89fbfc47794d92R522)

Reply:
Because of how cookiecutter parses and stores this, it becomes `"['None']"`, see: [other checks](https://github.com/kedro-org/kedro/pull/3462/files#diff-e3c3f6f5910e7f8d3133cfbeec750588c9328e0507c5464b6b89fbfc47794d92L852).

The reason L521 uses double quotes is due to linting and formatting

Reply:
Wow, unexpected that, thank you!